client/src/components/pages/homepage/style.css

```diff
@@ -0,0 +1,38 @@
+.home {
+  padding: 30px 16px 0 16px;
+  min-height: calc(100vh - (184px + 70px));
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+}
+
+#root {
```

Comment:
@IsraaSulaiman I might have mentioned this in another PR, but I think it might be better to put the #root styles into App.css (so you don't have to repeat them in every page's css file)